test/object-storage.test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var assert = require('assert');
+
+describe('object-storage connector', function() {
+  it('should be set up as loopback-component-storage', function(done) {
+    this.timeout(10000);
+    var app = require('./fixtures/object-storage-app');
+    var booted = false;
+    app.on('booted', function() {
+      if (booted) return;
+      assert('MyObjectStorage' in app.datasources);
```

Comment:
When this check fails, it will produce an error message that's not helpful. Here is a better solution:

```js
expect(Object.keys(app.datasources)).to.contain('MyObjectStorage');
```

(Or maybe `.to.have.member`? I am never certain about which Chai assertion to use for this.)